### ISSUE TITLE
defining the queryParamsDidChange action breaks refreshModel: true on query params

### ISSUE THREAD

**Issue by jesseditson** [emberjs/ember.js#14470]
I was investigating a similar issue, and ran across this scenario:

https://ember-twiddle.com/ea1da4a5972f74a6fb268deb254532b4

In the twiddle, there are 2 identical routes. On the `defined` route, we have a `queryParamsDidChange` action defined, with bubbling enabled. On the `undefined` route, we simply don't define the action. I'd expect the functionality of these two routes to be identical, but instead you can see that when updating the query params, the `defined` route never calls the model hooks, whereas the `undefined` route does.

**Comment by pixelhandler** [comment]
@jesseditson I'm curious about the http://emberjs.com/api/classes/Ember.Route.html#method_queryParamsDidChange action in your twiddle. That's also a name for a private method on the Route, should that name be avoided as an action in your route?

**Comment by jesseditson** [comment]
Looks like perhaps the docs miscategorized as a method (not sure what normal protocol around private events is) - I've seen this used elsewhere, to workaround double or early fires (the original issue I was troubleshooting), but understand that as a private method this may be expected behavior.

From the docs:
```
This action is called when one or more query params have changed. Bubbles.
```

I think perhaps this is just a documentation issue - I just did some exploring, and calling `this._super(...arguments)` causes this to "bubble" as I'd expect when returning `true`. This resolves my case, but I had to do some guessing to get there.

Let me know if I can help out clarifying the docs here, or if it's just my bad for poking around with private APIs 😛
